services/storehost/storehost_test.go

```diff
@@ -69,8 +84,8 @@ func (s *StoreHostSuite) TestStoreHostTimerQueueWriteWithRead() {
 		numExtents = 256                    // number of concurrent extents
 		minMessages, maxMessages = 0, 1000  // messages to send on an extent
 		minMsgSize, maxMsgSize = 512, 2048  // size of message
-		minMsgDelay, maxMsgDelay = 5, 300   // delay to use on messages (in seconds)
-		minRandWait, maxRandWait = 0, 10000 // wait between messages published (in milliseconds)
+		minMsgDelay, maxMsgDelay = 5, 30   // delay to use on messages (in seconds)
+		minRandWait, maxRandWait = 0, 1000 // wait between messages published (in milliseconds)
```

Comment:
these values were intended to apply to a super-long test that needs to be initiated manually .. (using the "-long" flag).